app/models/competition.rb

```diff
@@ -1058,19 +1054,11 @@ def country_zones
     self.longitude_microdegrees = @longitude_degrees * 1e6 unless @longitude_degrees.nil?
   end
 
-  def has_events_with_ids?(event_ids)
-    (event_ids - events.ids).empty?
-  end
-
-  def has_event?(event)
-    self.events.include?(event)
-  end
-
-  def has_base_entry_fee?
+  def base_entry_fee_set?
     base_entry_fee.nonzero?
   end
 
-  def has_fees?
+  def paid?
```

Comment:
Not a fan of this rename. `paid?` in the past tense implies that an action had happened which caused... the competition to now be considered as paid...??!

Reply:
well I was more thinking about free service vs. paid service.